AspNetScaffolding3Tests/Extensions/Logger/LoggerSettingsTest.cs

```diff
@@ -0,0 +1,82 @@
+using AspNetScaffolding.Extensions.Logger;
+
+using Xunit;
+
+namespace AspNetScaffolding3Tests.Extensions.Logger
+{
+    public class LoggerSettingsTest
+    {
+        private LoggerSettings _loggerSettings;
+
+        public LoggerSettingsTest()
+        {
+            _loggerSettings = new LoggerSettings();
+        }
+
+        [Fact]
+        public void GetInformationTitle_InformingInformationTitle_ReturnsInformedValue()
+        {
+            var informationTitle = "Information Title";
+            _loggerSettings.InformationTitle = informationTitle;
+
+            var expected = informationTitle;
+            var actual = _loggerSettings.GetInformationTitle();
+
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void GetInformationTitle_InformationTitleEqualNull_ReturnDefaundValue()
+        {
+            var expected = "HTTP {Method} {Path} from {Ip} responded {StatusCode} in {ElapsedMilliseconds} ms";
+            var actual = _loggerSettings.GetInformationTitle();
+
+            Assert.Equal(expected, actual);
+        }
+        
+        [Fact]
+        public void GetInformationTitle_InformationTitleEqualEmpty_ReturnDefaundValue()
```

Comment:
Default ta escrito errado.
```suggestion
        public void GetInformationTitle_InformationTitleEqualEmpty_ReturnDefaultValue()
```